phase-4-explainer.md

```diff
@@ -0,0 +1,175 @@
+### Phase 4: Full Introspection of an Accessibility Tree
+
+The **Computed Accessibility Tree** API will allow authors to access
```

Comment:
Thank you for using semantic line breaks 👍

Reply:
You're welks